server/backend/local-dev.env

```diff
@@ -4,9 +4,6 @@ POSTGRES_HOST=localhost
 POSTGRES_DB=fam
 POSTGRES_DB_TEST=fam_test
 POSTGRES_PORT=5432
-TMP_FLYWAY_DIR=../../server/flyway/sql
-api_db_username=fam_user
-api_db_password=fam_user_password
```

Comment:
Not sure if these "api_db_username" and "api_db_password" are safe to be removed from environment file; backend config.py seems to be reading that. Maybe this "api_db_username" should instead be of value: "fam_proxy_api"?

Reply:
TMP_FLYWAY_DIR is only used for the Alembic stuff that we are no longer using
api_db_username and api_db_password are indeed used in the config file, but it's still working without them because there are defaults. I'll add them back in for clarity.

Reply:
Actually, I was right. api_db_username and api_db_password are not used by that docker container.